### PR TITLE
Dimensions descriptions rewrite

### PR DESCRIPTION
**Pull request by boltzmann-brain**
Rewrote descriptions of each dimension to:

- remove old references to ART modules
- reflect new harness components

For Ethics and Privacy, I also started on adding a summary table at the top listing all probes, harnesses, and detectors associated with the dimension. Holding off on adding similar tables to other dimensions until we decide what the human-readable probe/detector/harness names are.

### CODE REVIEW

File: source/tests-library/ethics.md [hunk 1]
```diff
@@ -1,36 +1,33 @@
 # Ethics
 
-As intelligent agents designed to automate human-grade tasks, it is important for an LLM to have knowledge of basic concepts of morality that goes beyond toxicity, bias, or stereotypes— such as justice, well-being, duties, and virtues.
-
-We evaluate the propensity of an LLM to generate content not aligned with shared human values using 9 tests in 2 modules.
+| Harness                          | Probe                                                 | Detector                                              |
+| -------------------------------- | ----------------------------------------------------- | ----------------------------------------------------- |
+| vijil.harnesses.ethics           | All probes named below                                | autoredteam.detectors.base.TriggerListAbsenceDetector |
+| [vijil.harnesses.normative_ethics](#probes-from-the-ethics-dataset) | vijil.probes.normative_ethics.Deontology              | autoredteam.detectors.base.TriggerListAbsenceDetector |
```
Comment (boltzmann-brain) [inline source/tests-library/ethics.md:6]: will replace these with human-readable names when ready

File: source/tests-library/security.md [hunk 2]
```diff
@@ -61,38 +50,23 @@ Vijil's tests check for such indirect prompt injection attacks that use a variet
...
-    In our Python SDK, this test is covered by the module ``autoredteam.tests.glitch``.
-```
 
 ## Goodside
 
 This group of tests implements prompt injection attacks [found](https://simonwillison.net/2022/Sep/12/prompt-injection/)
 by AI security engieer Riley Goodside.
 
-Vijil covers 4 Goodside attacks through its tests that are intended to make the agent
+Vijil covers Goodside attacks through its tests that are intended to make the agent:
```
Comment (shubhobm) [inline source/tests-library/security.md:65]: we're dropping goodside right?

File: source/tests-library/hallucination.md [hunk 3]
```diff
@@ -35,9 +27,18 @@ packages gets infected.
...
 
-    This test is also part of our suite of Security tests.
-```
+These prompts attempt to get the model to generate false information about Riley Goodside. Common hallucinations involve claiming that Riley Goodside is in the entertainment industry, Canadian, Australian, or an entrepreneur.
+
+## Davidjl glitch
+
+These prompts attempt to get the model to repeat false information about the glitch token "Davidjl". They were [discovered](https://twitter.com/goodside/status/1666598580319035392) by Riley Goodside.
+
+## Deadnaming
```
Comment (shubhobm) [inline source/tests-library/hallucination.md:38]: this moves to toxicity?

File: source/tests-library/security.md
Comment (shubhobm) [inline source/tests-library/security.md:1]: threatjson moves under promptinject?

**Review by shubhobm — CHANGES_REQUESTED** [review]

File: source/tests-library/security.md (hunk 2 above)
Comment (boltzmann-brain) [reply source/tests-library/security.md:65]: No, all goodside probes are included they are just split across diff dimensions now. Some are in security.

File: source/tests-library/security.md
Comment (boltzmann-brain) [reply source/tests-library/security.md:1]: done

File: source/tests-library/hallucination.md (hunk 3 above)
Comment (boltzmann-brain) [reply source/tests-library/hallucination.md]: done

File: source/tests-library/security.md (hunk 2 above)
Comment (shubhobm) [reply source/tests-library/security.md]: ah got it

**Review by shubhobm — APPROVED** [review]
lgtm

**Pull request merged**